generator/beat/{beat}/main.go.tmpl

```diff
@@ -4,6 +4,7 @@ import (
 	"os"
 
 	"{beat_path}/cmd"
+	_ "{beat_path}/include"
```

Comment:
I think that unused imports should be separated into their own group and have a comment explaining it's existence. Similar to the [golint warning](https://github.com/golang/lint/blob/470b6b0bb3005eda157f0275e2e4895055396a81/lint.go#L458), but golint exempts `main` packages.